minimesos/src/main/java/com/containersol/minimesos/cluster/ClusterRepository.java

```diff
@@ -22,6 +22,7 @@
     /**
      * Loads representation of the running cluster
      *
+     * @param factory TODO(mwl)
```

Comment:
The point with javadoc is that it's detached from the source. So if you for some reason cannot see the source you can at least see it's documentation.